packages/core/src/components/card/card.tsx

```diff
@@ -27,9 +27,19 @@ export class Card {
    */
   @Prop() variant: CardVariant = 'insight';
 
+  /**
+   * @since 2.1.0
+   */
+  @Prop() selected: boolean;
+
   render() {
     return (
-      <Host class={`card-${this.variant}`}>
+      <Host
+        class={{
+          selected: this.selected,
+          [`card-${this.variant}`]: true,
```

Comment:
Just out of curiosity, why did you choose to use square brackets? If I recall correctly, it should lead to the same outcome.

Reply:
Without them it gives me a syntax error. I think it is necessary because of the template string.